app/services/state_file/state_information_service.rb

```diff
@@ -42,6 +50,27 @@ def state_code_from_intake_class(klass)
         end
         state_code.to_s
       end
+
+      def intake_class_from_state_code(state_code)
+        STATES_INFO[state_code][:intake_class]
+      end
+
+      def navigation_from_state_code(state_code)
+        STATES_INFO[state_code][:navigation]
+      end
+
+      def intake_classes
```

Comment:
this is redundant with `state_intake_classes` defined on line 27. if you like this name better that's fine but we should just pick one